components/output/sm2135.rst

```diff
@@ -59,6 +62,8 @@ Configuration variables:
 -  **data_pin** (**Required**, :ref:`Pin Schema <config-pin_schema>`): The pin used for MOSI.
 -  **clock_pin** (**Required**, :ref:`Pin Schema <config-pin_schema>`): The pin which SCLK is
    connected to.
+-  **rgb_current** (**Required**): Sets the current for the RGB LEDs (in milliamperes)
```

Comment:
Could we make this optional and set a sane default?

Reply:
the only sane default would be the minimum 10mA, i guess. and then the user could be wondering why the lights are so dim or not working at all. so i suggest to push the user to enter the correct values.